special_chars.h

```diff
@@ -1,18 +1,19 @@
-#ifndef SPECIAL_CHARS_H
-
-#define SPECIAL_CHARS_H
-
-#include <string>
-
-class Symbol{
-  public:
-    static std::string angstrom();
-    static std::string cubed();
-    static std::string subscript(int num); 
-    static std::string subscript(std::string num);
-    
-};
-
-#endif
-
-
+#ifndef SPECIAL_CHARS_H
+
+#define SPECIAL_CHARS_H
+
+#include <string>
+#include <cassert>

```

Comment:
I think the `cassert` here is necessary because I use `assert()` in the .cpp file. Maybe you can try moving the include into the cpp file and check whether it works?